docs/challenges.rst

```diff
@@ -16,18 +15,12 @@ design favors performing challenges automatically, and this is the normal case f
 
 Some plugins offer an *authenticator*, meaning that they can satisfy challenges:
 
-* Apache plugin: (TLS-SNI-01) Tries to edit your Apache configuration files to temporarily serve 
-  a Certbot-generated certificate for a specified name. Use the Apache plugin when you're running 
-  Certbot on a web server with Apache listening on port 443.
```

Comment:
Wow, guess we never updated this one! I'd say just change `TLS-SNI-01` to `HTTP-01` and `443` to `80` instead of deleting these.